Don't panic when get refresh rate failed (#2533)
rust-windowing/winit: CHANGELOG.md

- **Breaking:** Removed `WindowBuilderExtWindows::with_theme` and `WindowBuilderExtWayland::with_wayland_csd_theme` in favour of `WindowBuilder::with_theme`.
- **Breaking:** Removed `WindowExtWindows::theme` in favour of `Window::theme`.
- Enabled `doc_auto_cfg` when generating docs on docs.rs for feature labels.

> - Don't panic when get refresh rate failed, this usually happens when the monitor the window is on is removed.

Reviewer: ```suggestion
- On macOS and Windows, fixed crash when couldn't get monitor refresh rate.
```
